content/kubermatic/master/advanced/oidc_auth/_index.en.md

```diff
@@ -114,3 +114,29 @@ After all values are set at the `values.yaml` the installed helm charts `kuberma
 helm upgrade --install --wait --timeout 300 --values values.yaml --namespace oauth oauth charts/kubermatic/oauth
 helm upgrade --install --wait --timeout 300 --values values.yaml --namespace kubermatic kubermatic charts/kubermatic/
 ```
+
+### Role-Based Access Control Predefined Roles
+
+Kubermatic provides predefined Roles and ClusterRoles to help implement granular permissions for specific resources and to simplify access
+control across the user cluster. All of the default Roles and ClusterRoles are labeled with `component=userClusterRole`.
```

Comment:
ClusterRoles written with CamelCase looks strange in the docs. I'd write it as 2 words, perhaps even with lower case. This is for the user and this type is common, not Kubermatic-specific.